megatron/core/ssm/mamba_mixer.py

```diff
@@ -290,11 +290,11 @@ def __init__(
             )
             setattr(self.conv1d.weight, "tensor_model_parallel", True)
             setattr(self.conv1d.bias, "tensor_model_parallel", True)
-
-            if self.conv_init is not None:
-                nn.init.uniform_(self.conv1d.weight, -self.conv_init, self.conv_init)
-            else:
-                nn.init.kaiming_uniform_(self.conv1d.weight, a=math.sqrt(5))
+            if self.config.perform_initialization:
```

Comment:
Why do you want to disable initialization of these trainable variables (`conv1d.weight` and `A_log`)?

Also, I don't see `perform_initialization` in `TransformerConfig` at top-of-tree. Where is this set? Does it default to `True`?

cc @rogerwaleffe